### ISSUE TITLE
Tugas Pengantar Telekomunikasi

### ISSUE THREAD

**Issue by rahmatjuniansyah** [rahmatjuniansyah/rahmat-juniansyah#1]
troughput:
12158263/94.440 = 128,740 dalam bytes/s
1 bytes=8 bit
 = 128,740 x 8 =1029 kb/s

packet loss = (paket dikirim - paket yang diterima)/paket yang dikirim x 100
               (14432-14402)-14432)x100
            = 30 / 14432 x 100
            = 0,207

delay = 
  total delay = 94,439761
  rata-rata delay = 0,006557406 = 65,57406ms

jiter= 
    total jitter=94,465227 s
    rata-rata jitter=-14307,53477 s
